actionpack/lib/action_dispatch/middleware/actionable_exceptions.rb

```diff
@@ -19,6 +27,9 @@ def call(env)
       ActiveSupport::ActionableError.dispatch(request.params[:error].to_s.safe_constantize, request.params[:action])
 
       redirect_to request.params[:location]
+    rescue Exception => err
+      hooks.each { |hook| hook.call(err) }
```

Comment:
So the assumption is that hook will raise before we do below?

Reply:
Let's just say `error` instead of `err` throughout. Plenty of room for the full word 😄

Reply:
> So the assumption is that hook will raise before we do below?

Yeah. I'm not sure how clever we have to be here. 🤷‍♂️

> Let's just say error instead of err throughout. Plenty of room for the full word.

My Go habits are leaking here. 😅